tests/testthat/test-save_e61.R

```diff
@@ -704,4 +704,14 @@ test_that("Multi-panel graph examples", {
                sources = c("Sources", "Sauces"))))
   })
 
+  withr::with_tempdir({
+    expect_snapshot_file(suppressWarnings(
+      save_e61("plot-multi-1x2-1-long-panel-title.svg", p1_t, p2_t, pad_width = 3,
```

Comment:
You'll need a new file name because this is overwriting the snapshot from the previous test